about-mpdf/requirements-v7.md

```diff
@@ -6,7 +6,7 @@ permalink: /about-mpdf/requirements-v7.html
 modification_time: 2017-03-14T10:03:20+02:00
 ---
 
-mPDF 7.0 requires PHP `^5.6 || ~7.0.0 || ~7.1.0`. PHP `mbstring` (including `mbregex`, which needs to
+mPDF 7.0 requires PHP 5.6, 7.0.0 or 7.1.0. PHP `mbstring` (including `mbregex`, which needs to
```

Comment:
Mayba something along `PHP >= 5.6.0 and < 7.3.0`? (7.2.* should now be compatible)